src/main/java/org/generationcp/middleware/domain/study/StudyEntryListGeneratorRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package org.generationcp.middleware.domain.study;
+
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class StudyEntryListGeneratorRequestDto {
```

Comment:
@mderamos does it make sense to create a class only to have one attribute? Could not be possible for the service to receive an Integer instead of this class?